clustering/operations.go

```diff
@@ -241,7 +240,7 @@ func (p *managerProcess) failover(ctx context.Context, ss *StatusSet) error {
 
 	gtid := candidates[candidate].ReplicaStatus.RetrievedGtidSet
 	log.Info("waiting for the new primary to execute all retrieved transactions", "index", candidate, "gtid", gtid)
-	err = ss.DBOps[candidate].WaitForGTID(ctx, gtid, failOverTimeoutSeconds)
+	err = ss.DBOps[candidate].WaitForGTID(ctx, gtid, timeoutSeconds)
 	if err != nil {
 		return err
 	}
```

Comment:
I think we can loop this if `err` is `dbop.ErrTimeout`.
https://github.com/cybozu-go/moco/blob/main/pkg/dbop/replication.go#L69